easybuild/easyconfigs/l/libICE/libICE-1.0.9-GCC-11.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libICE'
+version = '1.0.9'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """X Inter-Client Exchange library for freedesktop.org"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+checksums = ['7812a824a66dd654c830d21982749b3b563d9c2dfe0b88b203cefc14a891edc0']
+
+builddependencies = [
+    ('xproto', '7.0.31'),
+    ('xorg-macros', '1.19.3'),
+    ('gawk', '5.1.1'),
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('xtrans', '1.3.5'),
+]
```

it looks like (at least, judging by the resultant installation of the `xtrans` easyconfig), that `xtrans` is header-only, so it is enough to provide it as a `builddependency`
```suggestion
    ('xtrans', '1.3.5'),
]
```

---

That makes sense. I was using an old EC as a template.